corehq/apps/accounting/migrations/0041_remove_like_indexes.py

```diff
@@ -0,0 +1,310 @@
+# encoding: utf-8
+import datetime
+from south.db import db
+from south.v2 import SchemaMigration
+from django.db import models
+
+class Migration(SchemaMigration):
+
+    def forwards(self, orm):
+        # db.execute("DROP INDEX IF EXISTS accounting_billingaccount_name_like")
+        db.execute("DROP INDEX IF EXISTS accounting_billingaccount_salesforce_account_id_like")
+        db.execute("DROP INDEX IF EXISTS accounting_billingaccountadmin_domain_like")
+        db.execute("DROP INDEX IF EXISTS accounting_billingaccountadmin_web_user_like")
+        db.execute("DROP INDEX IF EXISTS accounting_billingrecord_emailed_to_like")
+        db.execute("DROP INDEX IF EXISTS accounting_currency_code_like")
+        db.execute("DROP INDEX IF EXISTS accounting_currency_name_like")
+        db.execute("DROP INDEX IF EXISTS accounting_defaultproductplan_product_type_like")
+        db.execute("DROP INDEX IF EXISTS accounting_feature_feature_type_like")
+        # db.execute("DROP INDEX IF EXISTS accounting_feature_name_like")
+        db.execute("DROP INDEX IF EXISTS accounting_paymentmethod_method_type_like")
+        # db.execute("DROP INDEX IF EXISTS accounting_softwareplan_name_like")
+        # db.execute("DROP INDEX IF EXISTS accounting_softwareproduct_name_like")
+        db.execute("DROP INDEX IF EXISTS accounting_softwareproduct_product_type_like")
+        db.execute("DROP INDEX IF EXISTS accounting_subscriber_domain_like")
+        # db.execute("DROP INDEX IF EXISTS accounting_subscriber_organization_like")
+        # db.execute("DROP INDEX IF EXISTS accounting_wirebillingrecord_emailed_to_like")
```

Comment:
@biyeun @nickpell please review

The only 'like' queries I could find in the code were on `BillingAccount.name` and `SoftwarePlan.name`. Can I remove any of the other commented out ones?